io.kopia.KopiaUI.appdata.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>io.kopia.KopiaUI</id>
+  <name>KopiaUI</name>
+  <project_license>Apache-2.0</project_license>
+  <developer_name>Kopia Authors</developer_name>
+  <summary>Backup solution</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <url type="homepage">https://kopia.io/</url>
+  <url type="bugtracker">https://github.com/flathub/io.kopia.KopiaUI/issues</url>
+  <description>
+    <p>
+      Cross-platform backup tool for Windows, macOS &amp; Linux with fast, incremental backups, client-side end-to-end encryption, compression and data deduplication. CLI and GUI included.
+    </p>
```

Comment:
Please add a note here that this package is not supported by upstream, see https://flathub.org/apps/details/com.amazon.Workspaces for an example